### ISSUE TITLE
Index creation task not running

### ISSUE THREAD

**Issue by EmilePW** [meilisearch/meilisearch#2303]
**Describe the bug**
I have been using Meilisearch for a couple of months and have recently upgraded to 0.26.0. For some reason, when I am today trying to create an index using the node package, nothing seems to happen.

I can successfully use the createIndex method like below:

`client.createIndex("movies")`
and the return value for the method shows the task:

```
{
   uid: 26858,
   indexUid: 'movies',
   status: 'enqueued',
   type: 'indexCreation',
   enqueuedAt: '2022-04-08T15:15:06.325108519Z'
}
```
However, when I look up this task it seems that it has not been started:

```
{
  uid: 26858,
  indexUid: 'movies',
  status: 'enqueued',
  type: 'indexCreation',
  details: { primaryKey: null },
  duration: null,
  enqueuedAt: '2022-04-08T15:15:06.325108519Z',
  startedAt: null,
  finishedAt: null
}
```
And indeed I can't find the index using the getIndexes method.

Strangely I created an index without issue just a few days ago.

Any idea what the issue might be or how I could debug this?

**Expected behavior**
I expected the index to be created.

**Meilisearch version:** [e.g. v0.20.0]
0.26.0

**Additional context**
I have set up my Meilisearch instance by following the instructions for setting it up on AWS. I have also set up a custom domain. I don't think I have deviated from the guide in the docs.

**Comment by irevoire** [comment]
Hello, I don't know which version of meilisearch you were using before, but since the v0.25.x, the index creation became an asynchronous task. This means you need to wait until the task finish.

You can check the task status on the `/tasks/:taskUid` route.

I think your issue is that you have a lot of tasks already running, and thus your index creation has not been processed.

**Comment by EmilePW** [comment]
Okay turns out I had just assumed that the previous tasks had finished but when I looked back through them it appears that one of the index creations had been running for multiple days (and presumably stopping new tasks from being scheduled). I set up a new Meilisearch instance and used the `--enable-auto-batching` flag and reran that index creation which then completed much faster.

Not a bug - I just underestimated how long a task could be running for and didn't understand how the scheduling system worked.
